krypton/backend/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""This module exports krypton backend app callable."""
```

Comment:
Codacy found an issue: [First word of the docstring should not be `This` (D404)](https://app.codacy.com/manual/saivarunk/krypton/pullRequest?prid=5721332)